core/jazz_users/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jazz_users",
+  "version": "0.0.1",
+  "description": "Get the list of users from Amazon Cognito user pool",
+  "main": "index.js",
+  "private": true,
+  "author": {
+    "name": "Jazz"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "mocha": "^v3.0.0",
+    "aws-lambda-mock-context": "^3.1.1",
+    "aws-sdk-mock": "*",
+    "chai": "^3.5.0",
+    "sinon": "^5.0.7",
+    "aws-sdk": "*"
```

Comment:
aws-sdk should be a dependency instead of a dev dependency

Reply:
aws-sdk is installed globally for dependency.